quake-terminal@diegodario88.github.io/quake-mode.js

```diff
@@ -98,6 +98,28 @@ export const QuakeMode = class {
 		return actor;
 	}
 
+	get monitorDisplayScreenIndex() {
+		if (this._settings.get_boolean("render-on-current-monitor")) {
+			// current display
```

Comment:
@diegodario88 should I drop these little comments? overkill?

Reply:
Sure thing! Yes, you can drop those little comments.

Reply:
done